tests/unit/amo/pages/TestSearchTools.js

```diff
@@ -53,4 +55,43 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
+
+  describe('enableFeatureRemoveSearchTools = true', () => {
+    const renderWithoutSearchTools = (props = {}) => {
```

Comment:
I'm not sure of the value of creating `renderWithoutSearchTools` when it's only called once. You could probably just inline this code in the test.